### DISCUSSION TITLE
dp_ipi not in the forcefields list

### DISCUSSION THREAD

**Discussion by AxelTG** [deepmodeling/deepmd-kit#702] [Q&A] [answered]
Hi, I would like to carry out a simple PIMD simulation using dp_ipi. What forcefield should I indicate in the input.xml file? I tried with 'dp_ipi' but this error occurred: "Force component name 'dp_ipi' is not in the forcefields list". Any suggestions?

**Comment by njzjz** [comment, marked as answer]
https://github.com/deepmodeling/deepmd-kit/blob/ab7621e80302aeacbe67ae2767589dc66f840259/examples/water/ipi/input.xml#L22-L24
